internal/command/deploy/machines_launchinput.go

```diff
@@ -47,6 +47,9 @@ func (md *machineDeployment) launchInputForLaunch(processGroup string, guest *ap
 		mConfig.Standbys = standbyFor
 	}
 
+	// Merge files from the app config and the parsed flags.
+	machine.MergeFiles(mConfig, md.files)
```

Comment:
I kept going in circles trying to figure out how to merge files from flags and files from appconfig.Config before here and eventually gave up and went with this.

Reply:
I'd go with merging it at `determineAppConfigForMachines` because even if not the best, it is consistent with what we are doing now for other flags.

Reply:
Right, but that is what we do for env, strategy, region flags...  having a different treatment for files makes it worse.